### PR TITLE
Add support for terraform-lsp

### PR DESCRIPTION
**Pull request by jcmuller**

### CODE REVIEW

**Comment by lithammer** [comment]
Is there anything this is doing better than the official one? From what I can see this hasn't had any meaningful development in over a year.

**Comment by jcmuller** [comment]
Yes. They are different and complimentary. terraform-lsp still provides things that terraform-ls doesn't yet. For example: code completion.

File: lua/lspconfig/terraformlsp.lua [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    cmd = {"terraform-lsp"};
+    filetypes = {"terraform", "hcl"};
+    root_dir = util.root_pattern(".terraform", ".git");
+  };
+  docs = {
+    description = [[
+https://github.com/juliosueiras/terraform-lsp
+
+Terraform language server
+Download a released binary from https://github.com/hashicorp/terraform-ls/releases.
```
Comment (mjlbach) [inline lua/lspconfig/terraformlsp.lua:15]: You should copy the "does not modify your path" style warning etc. from elixirls

**Comment by mjlbach** [comment]
@mergifyio rebase

**Comment by mjlbach** [comment]
Can you double check this is passing style/commit lint? I know it is failing the latter.

**Comment by mjlbach** [comment]
I think terraform-ls has closed the gap with additional features, and seems to be more supported.

**Comment by mjlbach** [comment]
I'm going to close this, if there is interest in having this alongside terraform-ls in the future please fix the style lint and reopen

**Pull request closed without merging**